mappings/net/minecraft/sortme/OptionMainHand.mapping

```diff
@@ -1,3 +0,0 @@
-CLASS aie net/minecraft/sortme/OptionMainHand
```

Comment:
Where did this go?

Reply:
[`net/minecraft/client/options/MainHandOption`](https://github.com/FabricMC/yarn/pull/524/files#diff-bec61871facd317449b6b72fcf77d368)